crates/snapbox/src/data.rs

```diff
@@ -112,37 +112,25 @@ impl Data {
     ///
     /// Note: this will **not** do a binary-content check
     pub fn make_text(&mut self) -> Result<(), std::str::Utf8Error> {
-        *self = Self::text(std::mem::take(self).into_string()?);
+        *self = Self::text(std::mem::take(self).try_string()?);
```

Comment:
What was our plan for `make_test`, to remove it?  Let's have a PR do that and leave the current implementation as-is until then